orquesta/tests/unit/composition/native/test_workflow_basic.py

```diff
@@ -34,7 +34,7 @@ def test_sequential(self):
                     'id': 'task3'
                 },
                 {
-                    'id': 'noop'
+                    'id': 'continue'
```

Comment:
If we still support `noop`,  why not to crate a new test cases for `continue` ?

Reply:
The default task transition is being [changed](https://github.com/StackStorm/orquesta/pull/172/files#diff-bccb83fb04041ef611d45806a53c0825R86), so I think that this test needs to be changed.

However, unless I missed something, I do think this now needs tests for the `noop` transition.

Reply:
Agree, this exiting tests need to be changed for new `noop` behavior.

Reply:
There's already another unit test that covers `noop` at https://github.com/StackStorm/orquesta/blob/master/orquesta/tests/fixtures/workflows/native/join-count.yaml.  I will add another one here to be explicit.

Reply:
I've added a number of unit tests to cover the different uses of `continue`, `noop`, and `fail`.